src/flake8_tidy_imports/__init__.py

```diff
@@ -114,6 +139,36 @@ def rule_I250(
                         type(self),
                     )
 
+    @staticmethod
+    def compile_unstructured_glob(s: str) -> Pattern[str]:
```

Comment:
This is a rewrite of https://github.com/python/mypy/blob/b3ff2a683bd41a176c597d3c3ba65f6a31f805c5/mypy/options.py#L424-L432 to avoid full copy-pasting.